ironfish-cli/src/commands/miners/mined.ts

```diff
@@ -71,7 +82,12 @@ export class MinedCommand extends IronfishCommand {
         )
       }
 
-      progress.update(sequence - start)
+      speed.add(1)
+
+      progress.update(sequence - start, {
+        estimate: TimeUtils.renderEstimate(sequence - start, stop - start, speed.rate5s),
+        sequence: sequence,
```

Comment:
Trivial, but you could use object property value shorthand here and just do 
```suggestion
        sequence,
```

Reply:
I'm willing to agree to some type of standard but here's my general programming style for when I use that or not...

 - Use shorthand if all properties are shorthand
 - If there are any non shorthand properties, I expand all of them (usually)

I'm gonna merge this but let's keep the conversation going. What is your preference for using short hand construction?

Reply:
I think my preference is to spend less time redeclaring things when there's ambient variables you can soak up. I'm curious why you expand all of them if any are not-shorthand? For me each declaration is a cognitive step and I try to name things as I go so that I can ideally use shorthand everywhere, i.e.

```js
// rename from generic to specific in one
Object.fromEntries(blah).map(([contextSpecificName, value]) => ({contextSpecificName, value}))
// inline-rename
listOfObjects.map(({key: contextSpecificName, value}) => ({contextSpecificName, value}))
```

Reply:
Mainly the reason is traditional javascript reasons. Using tools to rename variables might inadvertently also change payloads without realizing it. It can still happen but it's less of a bug in typescript. Renaming foo below, would cause a bug if you also didn't realize this changes the API's payload returned.

```typescript
function request(res) {
    const foo = 2
    res.send({ foo })
}
```

In any case, I'm down to just prefer short hand versions. I'll update this PR and generally use that going forward if you have a preference.

Reply:
Updated